week2/index.js

```diff
@@ -1,65 +1,131 @@
+/*global console*/
+"use strict";
+// ##### GLOBAL VARIABLE DECLARATIONS #####
 var fs = require('fs');
-
 var options = process.argv.slice(2);
-
 var command = options[0];
-
+var valOfOptions = options[1];
+var dataOfTodos = openFile('todo.txt');
+var todos = splitStringByNewline(dataOfTodos);
+//console.log(todos);
+console.log(`
+######||***** welcome *****||######
+######|| My List Todos App ||######
+`);
+//This function is to read the file and catch the error 
+//Which we used it above To assign the variable we will use it later
+function openFile(fileName) {
```

Comment:
It's a really good practice to use helper functions like these.

Once you've written them, don't forget to use them! Try deleting the file `todo.txt` and then running `node index.js`. What happens? How can we fix this?